docs/b2b-edition/specs/api-v3/company/company.yaml

```diff
@@ -3052,7 +3081,452 @@ paths:
+                  data:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        companyId:
+                          type: integer
+                          description: ID of the company.
+                        companyName:
+                          type: string
+                          description: Name of the company.
+                        companyStatus:
+                          type: integer
+                          description: 'The status of the company. 0: PENDING 1: APPROVED 2: REJECTED 3: INACTIVE.'
+                          enum:
+                            - 0
+                            - 1
+                            - 2
+                            - 3
+                        subsidiaries:
+                          description: Subsidiaries of the company.
+                          type: array
+                          items:
+                            type: object
+                            properties:
+                              companyId:
+                                type: integer
+                                description: ID of the company.
+                              companyName:
+                                type: string
+                                description: Name of the company.
+                              companyStatus:
+                                type: integer
+                                description: 'The status of the company. 0: PENDING 1: APPROVED 2: REJECTED 3: INACTIVE.'
+                                enum:
+                                  - 0
+                                  - 1
+                                  - 2
+                                  - 3
+                              subsidiaries:
+                                type: array
+                                description: Subsidiaries of the company.
+                                items:
+                                  type: object
+                                  properties:
+                                    companyId:
+                                      type: integer
+                                      description: ID of the company.
+                                    companyName:
+                                      type: string
+                                      description: Name of the company.
+                                    companyStatus:
+                                      type: integer
+                                      description: 'The status of the company. 0: PENDING 1: APPROVED 2: REJECTED 3: INACTIVE.'
+                                      enum:
+                                        - 0
+                                        - 1
+                                        - 2
+                                        - 3
+                                    subsidiaries:
+                                      type: array
```

Comment:
This array requires an items sibling.
Is this structure correct? subsidiaries within subsidiaries?

Reply:
@bc-traciporter 
Yes, subsidiaries can also have subsidiaries. This is a schematic diagram of the company hierarchy.
<img width="1287" alt="image" src="https://github.com/user-attachments/assets/f0e940d5-e1a0-4a2d-a344-ae8e5a285417">